fe/fe-core/src/main/java/com/starrocks/load/Load.java

```diff
@@ -1203,5 +1204,41 @@ public long saveLoadJob(DataOutputStream dos, long checksum) throws IOException
 
         return checksum;
     }
+    public static class CSVOptions {
+        public String columnSeparator = "\t";
+        public String rowDelimiter = "\n";
+    }
+
+    public static TFileFormatType getFormatType(String fileFormat, String path) {
+        if (fileFormat != null) {
+            if (fileFormat.toLowerCase().equals("parquet")) {
+                return TFileFormatType.FORMAT_PARQUET;
+            } else if (fileFormat.toLowerCase().equals("orc")) {
+                return TFileFormatType.FORMAT_ORC;
+            } else if (fileFormat.toLowerCase().equals("json")) {
+                return TFileFormatType.FORMAT_JSON;
+            }
+            // Attention: The compression type of csv format is from the suffix of filename.
+        }
+
+        String lowerCasePath = path.toLowerCase();
+        if (lowerCasePath.endsWith(".parquet") || lowerCasePath.endsWith(".parq")) {
+            return TFileFormatType.FORMAT_PARQUET;
+        } else if (lowerCasePath.endsWith(".orc")) {
+            return TFileFormatType.FORMAT_ORC;
+        } else if (lowerCasePath.endsWith(".gz")) {
+            return TFileFormatType.FORMAT_CSV_GZ;
```

Comment:
We will support COMPRESSION for CSV, JSON, Avro and other file formats in the near steps, How can we support it?

Reply:
In my opinion, we can directly use FileFormat and compression type separately in FILES, regardless of it in Stream Load.

Reply:
Now, we get the compression type from the file name suffix. Maybe we could use a specialized property like `compression_type` to represent this.